test/extensions/wasm/wasm_speed_test.cc

```diff
@@ -0,0 +1,95 @@
+/**
+ * Simple WASM speed test.
+ *
+ * Run with:
+ * bazel run -c opt  --linkopt -fuse-ld=lld --define wasm=enabled --config=libc++ wasm_speed_test
```

Comment:
This should be really be:
```
TEST_SRCDIR=`pwd` TEST_WORKSPACE=bazel-$(basename `pwd`) bazel run --define wasm=enabled --config=libc++ -c opt //test/extensions/wasm:wasm_speed_test
```
1. You should really use the full path when describing Bazel targets.
2. `--linkopt -fuse-ld=lld` shouldn't be necessary (at least not universally).
3. `TEST_SRCDIR` and `TEST_WORKSPACE` are needed to run this.

Reply:
--linkopt -fuse-ld=lld is required on my box

If I include your definition of TEST_SRCDIR it can't find speed_cpp.wasm. I got the code for TEST_RUNDIR from envoy/test/main.cc.  What works for me is setting TEST_SRCDIR to the base envoy directory (i.e. where it was cloned).

I don't see how TEST_WORKSPACE can be "bazel-wasm" (TEST_WORKSPACE=bazel-$(basename `pwd`))  that directory doesn't appear anywhere in the workspace or in .cache/bazel

Reply:
Well, I stand corrected.  You solution works when I run from the root of the tree.

Reply:
Updated to the latest master.  Updated the run comments.  Left a note that lld might be required.

PTAL